docs/high-performance/Introduction.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Introduction to the High Performance package
+author: Sergio0694
+description: An overview of how to get started with the High Performance package and to the APIs it contains
+keywords: windows 10, uwp, windows community toolkit, uwp community toolkit, uwp toolkit, get started, visual studio, high performance, net core, net standard
+---
+
+# Introduction to the High Performance package
+
+The `CommunityToolkit.HighPerformance` contains helpers and extensions to work in high-performance scenarios. This package can be installed through NuGet, and it has the following multi-targets:
+
+- .NET Standard 2.0
+- .NET Standard 2.1
+- .NET Core 3.1
```

Comment:
.NET Core 3.1 is no longer supported eh? Didn't notice if this list was in the Diagnostics docs as well?

Reply:
Good catch! Fixed 🙂